modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationOutput.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gaul.annotation_processsor;
+
+import java.io.File;
+
+public final class ModernizerAnnotationOutput {
+
+    public static final String IGNORE_CLASSES_FILE_NAME =
+        "ignore-annotated-classes.txt";
+
+    private ModernizerAnnotationOutput() { }
+
+    public static File getOutputPath(File classOutputDir) {
+        if (classOutputDir.getPath().endsWith("/target/classes") ||
```

Comment:
Formatting nit (if their checkstyle accepts it). We put operators on leading lines, and will put the `) {` at the end of a multiline thing on its own line. So something like this would be standard here:

```java
if (classOutputDir.getPath().endsWith("/target/classes") 
    || classOutputDir.getPath().endsWith("/target/test-classes")
) {
```

Reply:
Their checkstyle complains saying - `OperatorWrap: '||' should be on the previous line.` :(

Reply:
:( If we can put the `) {` on the next line that'd still be helpful in seeing where the if ends and the code begins. If not we can put an empty line between the if and the code

Reply:
```
if (classOutputDir.getPath().endsWith("/target/classes") || 
    classOutputDir.getPath().endsWith("/target/test-classes")
    ) {
```
This works